grug/types/src/events/flat.rs

```diff
@@ -1,3 +1,9 @@
+#[cfg(feature = "sea-orm")]
+use sea_orm::entity::prelude::*;
+
+#[cfg(feature = "async-graphql")]
+use async_graphql::Enum;
+
 use {
```

Comment:
Can you remove the empty lines:

```suggestion
#[cfg(feature = "sea-orm")]
use sea_orm::entity::prelude::*;
#[cfg(feature = "async-graphql")]
use async_graphql::Enum;
use {
```